### DISCUSSION TITLE
Pregunta sobre el ejercicio de recolecta del dulces (Examen Parcial 1)

### DISCUSSION THREAD

**Discussion by marcof2une** [mmasias/24-25-PRG1#581] [Q&A] [answered]
Buenas @mmasias
Una consulta en conjunto con mi compañero Bienvenido Doalto
Cada vez que los niños salen de un edificio se vaciaria la bolsa? Porque al ejecutar el código con la manzana completa de edificios, solo suele llegar al edificio 2 o 3 como maximo ya que todos los niños  tienen  las bolsas llenas no visitan más los demás edificios y se termina el programa.

Como gustaria que abordemos este detalle Manuel?

**Comment by mmasias** [comment, marked as answer]
> Buenas @mmasias Una consulta en conjunto con mi compañero Bienvenido Doalto Cada vez que los niños salen de un edificio se vaciaria la bolsa? Porque al ejecutar el código con la manzana completa de edificios, solo suele llegar al edificio 2 o 3 como maximo ya que todos los niños tienen las bolsas llenas no visitan más los demás edificios y se termina el programa.
> 
> Como gustaria que abordemos este detalle Manuel?

Para este caso -y como han procedido correctamente, preguntando para afinar los requisitos-: lo dejo en sus manos: propongan lo que consideren más cercano a la realidad.

PS: en lo personal, pienso que los niños cuando se les llenan las bolsas dejan de pasear (no son codiciosos con los dulces). Además, así las demás casas tienen caramelos para otros niños.
